src/routes/Project/components/Table.js

```diff
@@ -215,10 +226,13 @@ export default class Table extends Component {
       </div>
     );
   }
+
+
 }
 
 Table.propTypes = {
-  numTracks: React.PropTypes.number.isRequired
+  numTracks: React.PropTypes.number.isRequired,
+  widthPx:   React.PropTypes.number.isRequired
```

Comment:
The state will eventually be synced with a database and with other users I dont think it should be included here, its not necessary, zooming can be handled with out adding it to the state.